x-pack/legacy/plugins/lens/public/app_plugin/plugin.tsx

```diff
@@ -33,22 +34,33 @@ export class AppPlugin {
     const datatableVisualization = datatableVisualizationSetup();
     const xyVisualization = xyVisualizationSetup();
     const metricVisualization = metricVisualizationSetup();
-    const editorFrame = editorFrameSetup();
-    const store = new SavedObjectIndexStore(chrome!.getSavedObjectsClient());
+    const editorFrameSetupInterface = editorFrameSetup();
+    this.store = new SavedObjectIndexStore(chrome!.getSavedObjectsClient());
```

Comment:
Since store is only used in `start` is there a reason it shouldn't just be a local const declared an initialized in the `start` method?